core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/provision/clusterproxy/ClusterProxyService.java

```diff
@@ -147,6 +156,16 @@ private String getAccountId(Stack stack) {
         return asList(gatewayTunnel, knoxTunnel);
     }
 
+    private List<CcmV2Config> ccmV2Configs(Stack stack) {
+        InstanceMetaData primaryGateway = stack.getPrimaryGatewayInstance();
+        return List.of(ServiceFamilies.GATEWAY.getDefaultPort(), ServiceFamilies.KNOX.getDefaultPort()).stream()
```

Comment:
If I remember well the decision was that there will be only one agent running of inverting proxy which would pass everything through nginx and it would mean we have to add a new context for knox in nginx. But in that case I don't think you need KNOX here.

Reply:
yes that was the approach decided. But I have not made any changes to serviceConfig which was generated and it is being commonly generated for ccmv1 and ccmv2.  For ccmv1 there is a service config entry `cmServiceConfig(stack, null, "cloudera-manager", clusterManagerUrl(stack))` , I was not sure if this is really not used for ccmv1.

Reply:
still, I think this won't work this way, as cluster proxy would try to connect to port 443, and that won't be available through CCMv2.
I don't how cluster proxy matches service configs with tunnel entries / ccmv2 configs, but I suspect you would need only one ccmv2 config while multiple service config, but a different one then for ccmv1

Reply:
while connecting to cluster CB uses **cb-internal** service endpoint which is mapped to 9443 port, so this doesnt break anything. For clarity I will create different service config for ccmv2.

Reply:
that's right, but other services depend on the other endpoint. Like `ClouderaManagerProxiedClientFactory` uses that connection inside CB.

Reply:
Removed KNOX from the config.